src/biokbase/narrative/jobs/job.py

```diff
@@ -240,6 +240,28 @@ def was_terminal(self):
         else:
             return self._acc_state.get("status") in TERMINAL_STATUSES
 
+    @property
+    def is_terminal(self):
+        self.state()
+        if self._acc_state.get("batch_job"):
+            for child_job in self.children:
+                if child_job._acc_state.get("status") != COMPLETED_STATUS:
+                    child_job.state(force_refresh=True)
+        return self.was_terminal
+
+    def in_cells(self, cell_ids):
+        """
+        For job initialization.
+        See if job is associated with present cells
+        """
+        if self.batch_job:
+            for child_job in self.children:
+                if child_job.cell_id in cell_ids:
+                    return True
+                return False
```

Comment:
Can you put in a comment to explain the behaviour with batch jobs?

What happens if the second job in `self.children` has a `cell_id` that isn't in `cell_ids`?

Reply:
Bug here